test/scripts/email-config.js

```diff
@@ -86,7 +86,12 @@ describe('scripts/email-config:', () => {
   })
 
   it('write fails if percentage is greater than 100', () => {
-    return cp.execAsync('echo \'{"sendgrid":{"percentage":101,"regex":".*"}}\' | node scripts/email-config write', { cwd })
+    return cp.execAsync('echo \'{"sendgrid":{"percentage":100.1,"regex":".*"}}\' | node scripts/email-config write', { cwd })
```

Comment:
Tweaked this because it occurred to me that an implementation which used `< 101` instead of `<= 100` would pass the old test but fail this new one.